### ISSUE TITLE
Wheels currently being built

### ISSUE THREAD

**Issue by martinhoyer** [SamuelYvon/netifaces-2#5]
README states that minimal python version is 3.5, but no cp35 or cp36 wheels are being built or at least are not available on PyPI.  
This results in netifaces2 not being able to be installed on el7 and el8 (python3.6):
```
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: /root/tst/bin/python3 /root/tst/lib64/python3.6/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-aby0ur0d/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'maturin>=0.13,<0.14'
       cwd: None
  Complete output (2 lines):
  ERROR: Could not find a version that satisfies the requirement maturin<0.14,>=0.13 (from versions: 0.7.1, 0.7.2, 0.7.6, 0.7.7, 0.7.8, 0.7.9, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.9.0, 0.9.1, 0.9.4, 0.10.0, 0.10.2, 0.10.3, 0.10.4, 0.10.5, 0.10.6, 0.11.0, 0.11.1, 0.11.2, 0.11.3, 0.11.4, 0.11.5, 0.12.0, 0.12.1, 0.12.2, 0.12.3, 0.12.4, 0.12.5, 0.12.6, 0.12.7, 0.12.8, 0.12.9, 0.12.10, 0.12.11, 0.12.12, 0.12.13, 0.12.14, 0.12.15, 0.12.16, 0.12.17, 0.12.18b1, 0.12.18b2, 0.12.18, 0.12.19, 0.12.20)
  ERROR: No matching distribution found for maturin<0.14,>=0.13
```

Perhaps it would be good to start using [cibuildwheel](https://github.com/pypa/cibuildwheel) to avoid headaches and also start using manylinux_2_28  (build manylinux2014 only for cp36).  

I can try to help with this if you want? :)

**Comment by SamuelYvon** [comment]
That's curious, I expected maturin to create all the wheels. It might be incompatible with those old versions? (I don't see why it would be the case). If you have time on your hands feel free to submit a PR. I'm out sick right now, but I'll review whatever comes my way.

**Comment by martinhoyer** [comment]
Thanks for the reply. I'm also sick atm, but will try to come up with something as soon as able.

**Comment by SamuelYvon** [comment]
Fixed in PR #6
